packages/belos/test/RealSolverManager/test_lsqr_complex.cpp

```diff
@@ -85,6 +85,13 @@ TEUCHOS_UNIT_TEST( LSQR, RealDoesNotThrow )
   TEST_NOTHROW( solver = rcp (new sol_mgr_type ()) );
 }
 
+// TODO: Reenable this test - Exception disabled for DII
+// I have temporarily removed the exception from the constructor to
+// demonstrate a working build wiht the new DII system. Currently all the
+// Tpetra classes will auto register complex. I need to discuss how we want
+// to handle this. Right now the DII system constructs and registers the
```

Comment:
I'm OK if we take the exception throw out of the constructors, as long as we leave it in the other methods.

Reply:
I restored this test to run but have it construct and then test for the throw on some arbitrary function. Currently using getNumIters().